app/controllers/web-payments/apple-pay/normalise-apple-pay-payload.js

```diff
@@ -15,19 +15,21 @@ const normaliseCardName = cardName => {
       throw new Error('Unrecognised card brand in Apple Pay payload: ' + cardName)
   }
 }
+
 const nullable = word => {
   if (word.length === 0 || !word.trim()) {
     return null
   }
   return word
 }
+
 const normaliseLastDigitsCardNumber = displayName => displayName.substr(displayName.length - 4)
 
 module.exports = payload => {
   const paymentInfo = {
     last_digits_card_number: normaliseLastDigitsCardNumber(payload.token.paymentMethod.displayName),
-    brand: normaliseCardName(payload.token.paymentMethod.network),
-    card_type: payload.token.paymentMethod.type.toLocaleUpperCase(),
+    brand: normaliseCardName(payload.token.paymentMethod.network.toLowerCase()),
```

Comment:
Would it be a good idea to move the `toLowerCase()` call into `normaliseCardName`? That way it wouldn't be dependent on the code that is calling it to know it has to make it lower case

Reply:
Yeah good point